debuglog-plugin/src/test/kotlin/com/bnorm/debug/log/IrPluginTest.kt

```diff
@@ -53,30 +65,146 @@ fun doSomething() {
 
   @Test
   fun `IR plugin enabled`() {
-    val result = compile(sourceFile = main, DebugLogComponentRegistrar(true))
-    assertEquals(KotlinCompilation.ExitCode.OK, result.exitCode)
+    testIrPlugin(
+      isPluginEnabled = true,
+      expectedGreetMethod = """
+        public static final String greet(@NotNull final String greeting, @NotNull final String name) {
+            Intrinsics.checkNotNullParameter(greeting, "greeting");
+            Intrinsics.checkNotNullParameter(name, "name");
+            System.out.println((Object)("⇢ greet(greeting=" + greeting + ", name=" + name + ')'));
+            final TimeMark markNow = TimeSource.Monotonic.INSTANCE.markNow();
+            try {
+                if (!Intrinsics.areEqual(greeting, "Hello")) {
+                    System.out.println((Object)("⇠ greet [" + (Object)Duration.toString-impl(markNow.elapsedNow-UwyO8pc()) + "] = " + "Early return"));
```

Comment:
Hmm... I wonder how brittle the name `elapsedNow-UwyO8pc` is going to be. I would think it needs to be constant to achieve binary compatibility with future versions of Kotlin, but it's just something interesting I noticed. No action, just musing.

Reply:
yeah. it is rather odd. Even the IntelliJ decompiler uses this name.